tasks/gulp/__tests__/check-individual-components-compile.test.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-env jest */
+
+const path = require('path')
+
+const sass = require('node-sass')
+
+const lib = require('../../../lib/file-helper')
+const configPaths = require('../../../config/paths.json')
+
+const sassRender = (options) => {
+  return new Promise((resolve, reject) => {
+    sass.render(options, (error, result) => {
+      if (error) {
+        return reject(error)
+      }
+      return resolve(result)
+    })
+  })
+}
+
+describe('Individual components', () => {
+  it('should compile individual scss files', done => {
```

Comment:
Can we have an expectation in this test (expect not to throw an exception?) to make it explicit what this is testing?